docs/examples/track_fusion_example.py

```diff
@@ -0,0 +1,564 @@
+
+PF_tracker_2 = general_tracker(SingleTargetTracker, radar2PF, PF_updater,
+                               PF_initiator, deleter, data_associator_PF)
+
+PartialTrackPF1, TrackFusionPF1 = tee(PF_tracker_1, 2)
+PartialTrackPF2, TrackFusionPF2 = tee(PF_tracker_2, 2)
+PartialTrackKF1, TrackFusionKF1 = tee(KF_tracker_1, 2)
+PartialTrackKF2, TrackFusionKF2 = tee(KF_tracker_2, 2)
+
+# Create the detector feeding the tracker algorithms
+track_fusion_tracker.detector = Tracks2GaussianDetectionFeeder(
+    MultiDataFeeder([TrackFusionKF1, TrackFusionKF2]))
+
+track_fusion_tracker2.detector = Tracks2GaussianDetectionFeeder(
+    MultiDataFeeder([TrackFusionPF1, TrackFusionPF2]))
+
+# Create a iterator for the trackers
+iter_fusion_tracker = iter(track_fusion_tracker)
+iter_fusion_tracker2 = iter(track_fusion_tracker2)
+
+# Loop over the timestep doing the track fusion and partial tracks
+for _ in range(number_of_steps):
+    for _ in range(2):
+        _, tracks = next(iter_fusion_tracker)
+        KF_fused_track.update(tracks)
+        del tracks
+        _, tracks = next(iter_fusion_tracker2)
+        PF_fused_track.update(tracks)
+
+    _, PF_sensor_track1 = next(iter(PartialTrackPF1))
+    PF_track1.update(PF_sensor_track1)
+
+    _, PF_sensor_track2 = next(iter(PartialTrackPF2))
+    PF_track2.update(PF_sensor_track2)
+
+    _, KF_sensor_track1 = next(iter(PartialTrackKF1))
+    KF_track1.update(KF_sensor_track1)
+
+    _, KF_sensor_track2 = next(iter(PartialTrackKF2))
+    KF_track2.update(KF_sensor_track2)
+
+truths = set(groundtruth_simulation.groundtruth_paths)
+
+# %%
+# Let's visualise the various tracks and detections in the cases
+# using the Kalman and particle filters.
+
+plotter.plot_tracks(PF_track1, [0, 2], line=dict(color="orange"), track_label='PF partial track 1')
+plotter.plot_tracks(PF_track2, [0, 2], line=dict(color="gold"), track_label='PF partial track 2')
+plotter.plot_tracks(PF_fused_track, [0, 2], line=dict(color="red"), track_label='PF fused track')
+plotter.plot_tracks(KF_fused_track, [0, 2], line=dict(color="blue"), track_label='KF fused track')
+plotter.plot_tracks(KF_track1, [0, 2], line=dict(color="cyan"), track_label='KF partial track 1')
+plotter.plot_tracks(KF_track2, [0, 2], line=dict(color="skyblue"), track_label='KF partial track 2')
+
+plotter.fig
+
+# %%
+# 4. Evaluate the obtained tracks with the groundtruth trajectory.
+# ----------------------------------------------------------------
+# At this stage we have almost completed our example. We have created the detections from the radars,
+# performed the tracking and the fusion of the tracks. We employ the :class:`~.MetricManager`
+# to generate summary statistics on the accuracy of the tracks by comparing them with the
+# groundtruth trajectory.
+#
+# If we consider the SIAP metrics, we can appreciate that the fused tracks have a lower error
+# compared to the partial tracks obtained with the single instruments.
+#
+
+# Instantiate the metric manager
```

Comment:
```suggestion
# Instantiate the basic metrics
```

Reply:
These are the "Basic Metrics" rather than the metric manager. You aren't actually outputting them here so you could remove lines 460-475, or follow the metrics example (https://stonesoup.readthedocs.io/en/latest/auto_examples/Metrics.html#) for how to output the results.